libs/sdk-backend-base/src/dummyBackend/index.ts

```diff
@@ -736,6 +738,14 @@ class DummyWorkspaceSettingsService implements IWorkspaceSettingsService {
     setLocale(_locale: string): Promise<void> {
```

Comment:
Please add an empty line to separate the definition of `getSettingsForCurrentUser` (line 725) from the function above it.